internal/librarian/generate.go

```diff
@@ -241,6 +245,41 @@ func (r *generateRunner) runBuildCommand(ctx context.Context, outputDir, library
 	return r.containerClient.Build(ctx, buildRequest)
 }
 
+// runConfigureCommand executes the container's "configure" command for an API.
+//
+// This step prepares a language repository for a new library by adding
+// necessary metadata or build configurations. It finds the library's ID
+// from the runner's state, gathers all paths and settings, and then delegates
+// the execution to the container client.
+func (r *generateRunner) runConfigureCommand(ctx context.Context) error {
+	apiRoot, err := filepath.Abs(r.cfg.Source)
+	if err != nil {
+		return err
+	}
+
+	// Configuration requires a language repository to modify. If one isn't specified or
+	// found, we cannot proceed.
+	if r.repo == nil {
+		slog.Info("No language repository specified; cannot run configure.", "api", r.cfg.API)
+		return errors.New("a language repository must be specified to run configure")
+	}
```

Comment:
nit: I believe if no repo specified, it should error our ealier in the process [here](https://github.com/cuiy0006/librarian/blob/8b713be11c28e467024b6827167b7b1a9b3f846b/internal/librarian/generate.go#L117-L120). 
Is this check necessary at this step? 
Also, if you want to log in addition to error, should you use slog.Error() instead?